locations/spiders/meijer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import ast
+from locations.items import GeojsonPointItem
+
+STATES = ['AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DC', 'DE', 'FL', 'GA',
+          'HI', 'ID', 'IL', 'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MD',
+          'MA', 'MI', 'MN', 'MS', 'MO', 'MT', 'NE', 'NV', 'NH', 'NJ',
+          'NM', 'NY', 'NC', 'ND', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC',
+          'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'WA', 'WV', 'WI', 'WY']
+
+class MeijerSpider(scrapy.Spider):
+    name = 'meijer'
+    allowed_domains = ['www.meijer.com']
+
+    def start_requests(self):
+        for state in STATES:
+            yield scrapy.Request(
+                'https://www.meijer.com/custserv/locate_store_by_state.cmd?form_state=locateStoreByStateForm&state={}'.format(state),
+                callback = self.parse
+            )
+    
+    def parse(self, response):
+        selector = scrapy.Selector(response)
+        stores = selector.css('div.records_inner>script::text').extract_first()
+        if stores:
+            for store in ast.literal_eval(stores.strip()[13:-1]):
```

Comment:
I'm not comfortable using `ast` here because it might accidentally execute code that we don't want it to. Instead of evaluating it into a Python list you might be able to parse it as JSON if you pre-process the text like so:

```python
stores = stores.replace('\',\'', '","')
stores = stores.replace('[\'', '["')
stores = stores.replace('\']', '"]')
stores = json.loads(stores)
```

Note that we're just replacing single quotes with double quotes.